Network/network-exchange-connectivity-matrix-configuration-guide.md

```diff
@@ -14,14 +14,18 @@ The table, below, documents the metropolitan areas where Network Exchange is off
 
 Metro Area|Colocation/<br>Dedicated Cloud Compute|Managed Hosting|CenturyLink Cloud
 ----------|------------------------------------|---------------|-----------------
-Ashburn, VA|DC2|DC2<br>DC3<br>DC4|VA1
+Ashburn, VA|DC2|DC2<br>DC3<br>DC4|VA1<br>VA2
 Santa Clara, CA|SC8|SC8<br>SC9|UC1
 Chicago, IL|CH3|CH3|IL1
-New Jersey|NJ2|NJ2<br>NJ2X|NY1
+New Jersey, NJ|NJ2|NJ2<br>NJ2X|NY1
+Seattle, WA|SE2|SE2|WA1
 London, UK|LO1|LO1|GB3
-Toronto, Canada|TR1|TR1|CA3
+London, UK|LO6|LO6|GB1
+Frankfurt, DE|FR6|FR6|DE1<br>DE3
+Toronto, Canada|TR1|TR1|CA3<br>CA2
+Sydney, AU|SY7|SY7|AU1
+Singapore, SG|SG2<br>SG8|SG1
```

Comment:
I think you missed an item table doesn't render properly.

should be Singapore, SG|SG2|SG2<br>SG8|SG1 I think based on your NJ item.